package.json

```diff
@@ -24,7 +24,14 @@
       "default": "./dist/js/uswds.min.js"
     },
     "./src/js/components": "./src/js/components/index.js",
-    "./src/js/components/*": "./src/js/components/*.js"
+    "./src/js/components/*": "./src/js/components/*.js",
```

Comment:
Based on your comment [here](https://github.com/uswds/uswds/issues/4507#issuecomment-1040781754) does that mean you still want this missing reference to be included?

Reply:
> this missing reference

If you're referring specifically to the Javascript reference, no, that's less urgent at the moment. It would be needed for reducing the amount of unused Javascript included in our bundles (manual tree shaking), but not impacting our ability to get things built/shipped.

It's primarily the image and font assets that this is needed for.